bakery/oven.go

```diff
@@ -240,12 +240,12 @@ func (o *Oven) NewMacaroon(ctxt context.Context, version macaroon.Version, expir
 }
 
 // AddCaveat adds a caveat to the given macaroon.
-func (o *Oven) AddCaveat(ctxt context.Context, m *macaroon.Macaroon, cav checkers.Caveat) error {
-	return AddCaveat(ctxt, o.p.Key, o.p.Locator, m, cav, o.p.Namespace)
+func (o *Oven) AddCaveat(ctx context.Context, m *Macaroon, cav checkers.Caveat) error {
+	return m.AddCaveat(ctx, cav, o.p.Key, o.p.Locator)
 }
 
 // AddCaveats uses o.AddCaveat to add all the given caveats to the macaroon.
-func (o *Oven) AddCaveats(ctxt context.Context, m *macaroon.Macaroon, caveats []checkers.Caveat) error {
+func (o *Oven) AddCaveats(ctxt context.Context, m *Macaroon, caveats []checkers.Caveat) error {
 	for _, cav := range caveats {
```

Comment:
Is there any reason not to use m.AddCaveats to do this now? (other than the comment would be lying)